dessia_common/datatools/dataset.py

```diff
@@ -671,8 +671,10 @@ def _tooltip_attributes(self):
 
     def _plot_data_list(self):
```

Comment:
Same remarks than `cluster` here

Reply:
Well, on second thought, this be of code could even be mutualized between the two of them

Reply:
Done